Cargo.toml

```diff
@@ -162,6 +162,14 @@ xml5ever = "0.20"
 opt-level = 3
 debug-assertions = true
 
+# Servo is slow in debug mode, so for daily testing we provide a devel profile
+# which is optimized, but has incremental building enabled for faster compile times
+[profile.devel]
```

Comment:
this name is too similar to cargo’s `dev` (which is already a confusing can of worms due to the whole `[profile.dev]` vs `target/debug` thing). what about something like `dev-fast`?